README.md

```diff
@@ -2,29 +2,29 @@
 
 Parsy is an abbreviation of `Polder Knowledge Pattern Library`. Combined with the 
 latin word for component `pars`. This repository contains the definitions of the 
-patterns that we have approved.
+components and patterns that we have approved to the Parsy library.
 
-## What is a pattern
+## Component and patterns you say?
 
-In Parsy terms a pattern is a filed template. Just a page of documentation how a common UX/UI
-problem should be solved. We believe that a basic pattern is very small. The more complex patterns
-are always a composite of smaller patterns. In extreme you can state that a html element is a kind of
-pattern to solve a common UI problem.
+We make a difference between components and patterns. In Parsy terms a component is a solution to a visual problem.
+Examples of components are buttons, form controls, tables, tab controls, etc. This solution we have documented. We 
+believe that components are very small. When one combines components to solve UI/UX problems, we call that a pattern.
+Patterns are always a composite of smaller components.
 
-A pattern doesn't need to have a style. In some situations it might be useful to have default colors
-and font sizes but it is up to the user of the pattern to decide if that should be changed or not. Inspired
-by [empties][empties-bourbon] by bourbon.
+A component doesn't need to have a style. In some situations it might be useful to have default colors
+and font sizes but it is up to the end-user to decide if that should be changed or not. Inspired by 
+[empties][empties-bourbon] by bourbon.
 
-A pattern can have one or multiple implementations. We believe that a pattern should solve a problem without forcing
-you to use a certain framework. Therefor a pattern starts with a good description of the goal of the pattern. If this
- solves your problem you can either use an existing implementation or create your own. In some eco-systems it is not 
- easy to adopt a new framework or library. You should be able to choose!
+A component can have one or multiple implementations. We believe that a pattern should solve a problem without forcing
+you to use a certain framework. Therefor a component starts with a good description of the goal of the component. If 
+this solves your problem you can either use an existing implementation or create your own. In some eco-systems it is not 
+easy to adopt a new framework or library. With Parsy we want to give you a choice!
  
 ## Why?
 
 We are sure that we are solving the same UX/UI problems every day. Sometimes these problems relate to basic patterns 
-and other times they are more complex. Therefor we believe in a library like this to describe these problems
- so we are able to recognize them and choose for an existing implementation before writing our own.
+and other times they are more complex. Therefor we believe in a library, like this, to describe these problems so we 
```

Comment:
Double comma's in this sentence reads a bit strange. I think the comma after libray is obsolete,